pytest_html/plugin.py

```diff
@@ -254,6 +258,7 @@ def append_extra_html(self, extra, extra_index, test_index):
                         class_=extra.get("format"),
                         href=href,
                         target="_blank",
+                        download=download,
```

Comment:
For non self-contained HTML this becomes `download="None"`. You probably need to add the argument conditionally somehow.

Reply:
```python
kwargs = {
    "class_": extra.get("format"),
    "href": href,
    "target": "_blank",
}

if download:
    kwargs["download"] = download

href = self.create_asset(
    content, extra_index, test_index, extra.get("extension"), **kwargs,
)
```

Reply:
Done (I think)